benchmarks/hash_table/dynamic_map/contains_bench.cu

```diff
@@ -18,7 +18,7 @@
 #include <utils.hpp>
 
 #include <cuco/dynamic_map.cuh>
-#include <cuco/utility/key_generator.hpp>
+#include <cuco/utility/key_generator.cuh>
```

Comment:
Oh noes, this might have been part of the culprit. A .hpp file with device code/lambdas in it.